src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -76,14 +76,14 @@ class Artwork extends React.Component<Props, any> {
         <Badges>
           {is_acquireable && (
             <Badge>
-              <Sans fontSize={8} lineHeight={8} style={{ paddingTop: 1 }} weight="medium" size="1">
+              <Sans fontSize="8px" lineHeight={8} style={{ paddingTop: 1 }} weight="medium" size="1">
```

Comment:
Without the `px` hardcoding here, the values appearing in the snapshot changed dramatically from `8` to something like `72`. I think  this was the result of the issue described in https://github.com/artsy/reaction/issues/1535.

Reply:
Hmm, interesting... is it only affecting the snapshot or is the line height also affected in the UI?

Reply:
I think this is related to @mzikherman updating the badges to include an item that was outside of Palette's theme file, per @briansw's request.

Reply:
It affects both:

Before fixing `px`:
<img width="151" alt="screen shot 2018-11-20 at 2 18 08 pm" src="https://user-images.githubusercontent.com/5216744/48797091-4da75f00-eccf-11e8-9420-e6dfc1355695.png">

After:
<img width="164" alt="screen shot 2018-11-20 at 2 19 00 pm" src="https://user-images.githubusercontent.com/5216744/48797090-4da75f00-eccf-11e8-842a-719e19e79528.png">

My understanding of the issue is that it only affects spots where we're passing values into `fontSize` and `lineHeight` overrides like this, though this makes me concerned about us potentially doing this in spots we don't cover with snapshot tests - I'll see if I can find any other affected spots with a search.

Reply:
@javamonn - would you mind raising this in #front-end?